Core/CoreTests/Inbox/Stubs/URLStub.swift

```diff
@@ -0,0 +1,30 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2024-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import Foundation
+
+enum FileStub {
+   static func createTemporaryFile(with size: Int) -> URL? {
+        let tempDirectory = FileManager.default.temporaryDirectory
+        let tempFileURL = tempDirectory.appendingPathComponent(UUID().uuidString)
+        let dummyData =  Data(repeating: 0, count: size)
+        try? dummyData.write(to: tempFileURL)
+        tempFileURL.stopAccessingSecurityScopedResource()
```

Comment:
No, must use stopAccessingSecurityScopedResource() function so can test case pass the code, because the code make security  checker and i can't mock it 

func addFiles(urls: [URL]) {
        urls.forEach { url in
            if url.startAccessingSecurityScopedResource() {
                addFile(url: url)
            }
        }
    }